src/components-examples/material/sidenav/sidenav-harness/sidenav-harness-example.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {TestBed, ComponentFixture} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {
+  MatDrawerHarness,
+  MatDrawerContainerHarness,
+  MatDrawerContentHarness} from '@angular/material/sidenav/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {
+  BrowserDynamicTestingModule,
+  platformBrowserDynamicTesting,
+} from '@angular/platform-browser-dynamic/testing';
+import {MatSidenavModule} from '@angular/material/sidenav';
+import {SidenavHarnessExample} from './sidenav-harness-example';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+
+describe('DrawerHarnessExample', () => {
```

Comment:
This says that it's the drawer harness, but we're using the sidenav harness everywhere.